sixtools/plotting_functions.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+
+def plot_frame(ax, frame, light_ROIs=[], cax=None, **kwargs):
```

Comment:
Although these plotting functions look OK too me it might be worth having @tacaswell take a look, any comments Tom?

Reply:
Any comments on these function would be much appreciated, but to some extent the user can just plot the result directly if they don't like the function. 

It would be more useful if you @tacaswell could look at
example_notesbooks/Analyze_data.ipynb